src/common/home/Sponsors.jsx

```diff
@@ -0,0 +1,157 @@
+      logo: '/sponsors-partners/nhost.png'
+    },
+    {
+      name: 'stackstream',
+      website: 'https://stack-stream.com',
+      logo: '/sponsors-partners/stackstream.png'
+    }
+  ];
+  const eventPartners = [
+    {
+      name: 'React Nexus',
+      website: 'https://reactnexus.com',
+      logo: '/sponsors-partners/react-nexus.png'
+    },
+    {
+      name: 'React Summit',
+      website: 'https://reactsummit.com/',
+      logo: '/sponsors-partners/react-summit.png'
+    }
+  ];
+
+  return (
+    <>
+      <h2 className="title-primary">
+        <strong>
+          <span>Our Partners & Sponsors</span>
+        </strong>
+      </h2>
+      <div className="event-partners-sponsors-container">
+        <div className="event-partners-sponsors">
+          <h3>Event Partners</h3>
+          <div className="content">
+            <div className="partners-sponsors-grid">
+              {eventPartners.map((p) => (
+                <a
+                  className="partners-sponsors-grid-item"
+                  href={p.website}
+                  rel="noreferrer"
+                  target="_blank"
+                >
+                  <img alt={p.name} src={p.logo} />
+                </a>
+              ))}
+            </div>
+            <a
+              className="partners-sponsors-cta"
+              href="https://forms.gle/19KTxhzp4aiTJ4kMA"
+              rel="noreferrer"
+              target="_blank"
+            >
+              Partner an Event <FaHandshake className="handshake" size={28} />
+            </a>
+          </div>
+        </div>
+        <div className="event-partners-sponsors">
+          <h3>Event Sponsors</h3>
+          <div className="content">
+            <div className="partners-sponsors-grid">
+              {eventSponsors.map((s) => (
+                <a
+                  className="partners-sponsors-grid-item"
+                  href={s.website}
+                  rel="noreferrer"
+                  target="_blank"
+                >
+                  <img alt={s.name} src={s.logo} />
+                </a>
+              ))}
+            </div>
+            <a
+              className="partners-sponsors-cta"
+              href="https://forms.gle/19KTxhzp4aiTJ4kMA"
+              rel="noreferrer"
+              target="_blank"
+            >
+              Sponsor an Event <FaHandshake className="handshake" size={28} />
+            </a>
+          </div>
+        </div>
+      </div>
+      {/* NOTE: In the following two sections, there are inline styles to override the custom css classes. These are being used for the time being, until data for the sections are available. */}
+      <div className="backers-container">
+        <h3>Backers</h3>
+        <span>
+          Support us with any monthly sponsorship. Your help means a lot to us.
+          <br /> We will give you a shoutout and add you here.
+        </span>
+        <div
+          className="backers-grid"
+          style={{
```

Comment:
Why inline styles?

Reply:
To override the default CSS, which was originally planned for having sponsors. Since we don't have any yet, I'm overriding the styles here.